apps/heft/src/schemas/typescript.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "TypeScript Build Configuration",
+  "description": "Defines optional options for TypeScript build.",
```

Comment:
Is this a declaration of `tsconfig.json`? Or is it proprietary to Heft? We should document that more clearly.

Also, how do I use it?  `heft-test-01` doesn't seem to have any config file conforming to this schema.

Reply:
No, this file provides some heft-specific options. I'll add usage of it to `heft-test-01`